core/mesos-master-dynamic/templates/mesos-master.sysconfig

```diff
@@ -1,30 +1,27 @@
-# common configuration
+# defaults
+MESOS_EXTERNAL_LOG_FILE=/var/log/mesos
 MESOS_FIREWALL_RULES=file:///etc/sysconfig/mesos-master-firewall-rules.json
+MESOS_LOGGING_LEVEL="INFO"
 MESOS_LOG_DIR=/var/log/mesos
-MESOS_LOGGING_LEVEL={{or (key "mesos/master/logging_level") "INFO"}}
-{{with key (plugin "keyForNode" "mesos/masters/%s/ip")}}MESOS_IP={{.}}{{else}}# MESOS_IP={{end}}
-MESOS_PORT={{or (key (plugin "keyForNode" "mesos/masters/%s/port")) 5050}}
-MESOS_EXTERNAL_LOG_FILE=/var/log/mesos
+MESOS_PORT=5050
+MESOS_WORK_DIR=/var/lib/mesos
+MESOS_ZK=zk://localhost:2181/mesos
```

Comment:
Yeah, this makes sense since we’ll be using Consul at this point. 
I’ll change it.